src/main/java/Duke.java

```diff
@@ -1,10 +1,258 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
+/**
+ * Duke is a chat bot which can help the user do multiple tasks
+ */
 public class Duke {
+    //Identifiers and values
+    public static final int ARRAY_SIZE_TASKS = 100;
+    public static final String DEADLINE_IDENTIFIER = "/by";
+    public static final String EVENT_IDENTIFIER = "/at";
+    public static final String WHITESPACE_IDENTIFIER = " ";
+    public static final String COMMAND_TODO = "todo";
+    public static final String COMMAND_DEADLINE = "deadline";
+    public static final String COMMAND_EVENT = "event";
+    public static final String REPLACEMENT_BLANK_SPACES = "";
+
+    //Printed Messages
+    public static final String MESSAGE_DOUBLE_WHITESPACE = "   ";
+    public static final String MESSAGE_LINE_SEPARATOR =
+            "______________________________________________________________________________";
+    public static final String MESSAGE_NOW_YOU_HAVE = " Now you have ";
+    public static final String MESSAGE_IN_THE_LIST = " in the list.";
+    public static final String MESSAGE_ERROR_TASK_UNAVAILABLE = "No tasks available... (｡◕‿‿◕｡)";
+    public static final String MESSAGE_TASKS_IN_LIST = " Here are the tasks in your list: ";
+    public static final String MESSAGE_ERROR_INVALID_COMMAND = "Invalid command entered... (ㆆ _ ㆆ)";
+    public static final String MESSAGE_ERROR_INVALID_NUMBER = "Invalid task number entered... (ㆆ _ ㆆ)";
+    public static final String MESSAGE_TASK_MARKED = "NICE! (｡◕‿‿◕｡) I've marked the task as done!:";
+    public static final String MESSAGE_TASK_ADDED_CONFIRM = " Got it, I've added this task: ";
+    public static final String MESSAGE_HELLO_FROM = "Hello from\n";
+    public static final String MESSAGE_INTRO_GREETING = " Hello! I'm Walter  ◕_◕";
+    public static final String MESSAGE_INTRO_WALTER_QUERY = " What can I do for you?";
+    public static final String MESSAGE_CLOSING = " I'm sad to see you go. Hope to see you again soon! :D";
```

Comment:
I like how you declare your messages as constants with appropriate naming practices.